dockerfiles/vcpkg/build.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+################################################################################
+# Script should be run from HPCC-Platform source directory
+# Will attempt to pull hpccbuilds/vcpkg-<target>:<VCPKG_REF> from docker.io
```

Comment:
This line will be invalid very shortly - I would either remove or replace with "This script is primarily for development when working on the Dockerfiles." or similar

Reply:
Removed entirely since it's changing in the future.